src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+$size-stopwatch: 80vmin;
+$size-border: 1vmin;
+$minute-rotate: 60s;
+
+.page__body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.stopwatch {
+  $minute-rotate: 60s;
```

Comment:
The variable `$minute-rotate` is redefined within the `.stopwatch` class. This redefinition is unnecessary and could lead to confusion. Consider removing it to maintain clarity.